### ISSUE TITLE
Quest: The Big Bone Worm [10930]

### ISSUE THREAD

**Issue by aqsdcv725xxp** [dalaranwow/dalaran-wow#4726]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Quest: http://www.wowhead.com/quest=10930
Fumper (Drum): http://www.wowhead.com/item=31825
Sand Gnome: http://www.wowhead.com/npc=22483
Hai'shulud: http://www.wowhead.com/npc=22038
Mature Bone Sifter: http://www.wowhead.com/npc=22482

**Expected behaviour**: Tell us what should happen instead

Problem:
Using the fumper near a clefthoof corpse just removes corpse and nothing else. Tried this on numerous clefthoofs with no joy.

**Steps to reproduce the problem**:

1. Take quest and find a Clefthoof
2. Kill the Clefthoof and use the drum near the corpse
3. By using the drum near the corpse, you should spawn either 3 Sand Gnomes, 2 Mature  Bone Sifter or the Hai'shulud himself. If any of first 2 spawn, kill them, kill more clefthoofs and try again. If "boss"  spawns, kill him, loot his organs and turn in the quest.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by ashido96** [comment]
#4256

**Comment by Rushor** [comment]
As you can see the issue was already reported in https://github.com/dalaranwow/dalaran-wow/issues/4256.

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
